lib/openra.rb

```diff
@@ -1,7 +1,7 @@
 DOWNLOAD_GITHUB_BASE_PATH = "https://github.com/OpenRA/OpenRA/"
 
 # Disable if you're doing something that may hit the gh rate limit (60 queries per hour for non-authenticated users)
-ENABLE_GITHUB_API = true
+ENABLE_GITHUB_API = false
```

Comment:
I turned this off because I was working on it when I had no internet – it should stay on for the release!